src/Database/Esqueleto/Experimental.hs

```diff
@@ -46,44 +61,95 @@ module Database.Esqueleto.Experimental
     , ToAliasReference(..)
     , ToAliasReferenceT
     -- * The Normal Stuff
-    , module Database.Esqueleto
-    )
+    , where_, groupBy, orderBy, rand, asc, desc, limit, offset
+             , distinct, distinctOn, don, distinctOnOrderBy, having, locking
+             , sub_select, (^.), (?.)
+             , val, isNothing, just, nothing, joinV, withNonNull
+             , countRows, count, countDistinct
+             , not_, (==.), (>=.), (>.), (<=.), (<.), (!=.), (&&.), (||.)
+             , between, (+.), (-.), (/.), (*.)
+             , random_, round_, ceiling_, floor_
+             , min_, max_, sum_, avg_, castNum, castNumM
+             , coalesce, coalesceDefault
+             , lower_, upper_, trim_, ltrim_, rtrim_, length_, left_, right_
+             , like, ilike, (%), concat_, (++.), castString
+             , subList_select, valList, justList
+             , in_, notIn, exists, notExists
+             , set, (=.), (+=.), (-=.), (*=.), (/=.)
+             , case_, toBaseId
```

Comment:
This block has some weird indentation; is that intentional or a copy/paste formatting error from another module?

Reply:
Looks like copy pasta

Reply:
I copied this directly from `Database.Esqueleto`. I think the original author's reasoning was that these are more or less all operators that go inside of a `where_`. The additional indent can be removed but I don't recommend flattening the entire list, as that would add like 60 lines to the export list.

Reply:
That's from when the class used to be a thing, and it was explicitly exported, so it was like

```
module Database.Esqueleto 
  ( class Esquleto ( subList_select, in_, notIn, exists, notExists, ...)
  ,   -- etc
  )
```

Rather than flattening the export list I just deleted the `class Esqueleto` bit and surrounding parens.
Laziness more than any principled thing.

Reply:
this will be fixed with #218